api4/user_test.go

```diff
@@ -2577,214 +2473,511 @@ func TestSwitchAccount(t *testing.T) {
 		NewPassword:    th.BasicUser.Password,
 	}
 
-	_, resp = Client.SwitchAccountType(sr)
+	_, resp = th.Client.SwitchAccountType(sr)
+	CheckUnauthorizedStatus(t, resp)
+}
+
+func assertToken(t *testing.T, th *TestHelper, token *model.UserAccessToken, expectedUserId string) {
```

Comment:
Would adding `t.Helper()` to this method help improve readability if the tests were to fail since it causes the stack trace printed to point to the line where `assertToken` is called instead of the line inside this function that failed